src/rgw/rgw_user.cc

```diff
@@ -2847,6 +2847,37 @@ int RGWUserCtl::list_buckets(const rgw_user& user,
   });
 }
 
+int RGWUserCtl::list_buckets(const rgw_user& user,
+                             const string& marker,
+                             const string& end_marker,
+                             uint64_t max,
+                             bool need_stats,
+                             RGWUserBuckets *buckets,
+                             bool *is_truncated,
+                             const Span& parent_span,
+                             uint64_t default_max)
+{
+  Span span_1 = tracer_2.child_span("rgw_bucket.cc : RGWUserCtl::list_buckets", parent_span);
```

Comment:
can you explain why tracer_2? can you rename it to something more meaningful and we would like to keep 1 tracer instance throughout.

Reply:
noted